scripts/scripts.js

```diff
@@ -69,7 +69,7 @@ const _addResultAsync = async (track, year, month, day, race, results) => {
     Horse.abi,
     process.env.CONTRACT_ADDRESS
   );
-  const _nm = bytes32({ input: track });
+  const _nm = bytes16({ input: track });
```

Comment:
should this be `formatBytes16String()`?

Reply:
Actually, I don't think this script is used.

Reply:
Yeah, scripts can be removed.